products/BellHybrid/apps/application-bell-meditation-timer/models/Statistics.cpp

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2017-2022, Mudita Sp. z.o.o. All rights reserved.
+// For licensing, see https://github.com/mudita/MuditaOS/LICENSE.md
+
+#include "Statistics.hpp"
+#include <ApplicationCommon.hpp>
+#include <db/ServiceDB.hpp>
+#include <db/MeditationStatsMessages.hpp>
+
+namespace
+{
+    using namespace service::db::meditation_stats;
+    std::optional<messages::Response> sendDBRequest(sys::Service *serv, std::shared_ptr<sys::Message> &&msg)
+    {
+        const auto ret = serv->bus.sendUnicastSync(msg, service::name::db, sys::BusProxy::defaultTimeout);
```

Comment:
to benefit from rvalue is needed std::move(msg)

Reply:
True:)